wormhole-connect/tests/ci/sdkv2.test.ts

```diff
@@ -213,35 +213,35 @@ describe('token', () => {
   }
 });
 
-describe('compare isSupportedChain between v1 and v2 routes', () => {
-  const compareChains = (network: NetworkV1) => {
-    const chains = chainLists[network];
-
-    for (const chain of chains) {
-      for (let [RouteV1, RouteV2] of routeMappings) {
-        if (
-          (chain === 'evmos' || chain == 'osmosis' || chain === 'kujira') &&
-          (RouteV2.meta.name === 'ManualTokenBridge' ||
-            RouteV2.meta.name === 'AutomaticTokenBridge')
-        ) {
-          // Temp hack. These are treated differently in SDKv2
-          continue;
-        }
-
-        test(`${RouteV1.name} (v1) vs. ${RouteV2.meta.name} (v2) - isSupportedChain(${chain})`, () => {
-          const v1Route = new RouteV1();
-          const v2Route = new SDKv2Route(network, RouteV2);
-          const isSupportedV1 = v1Route.isSupportedChain(chain);
-          const isSupportedV2 = v2Route.isSupportedChain(chain);
-          expect(isSupportedV1).toEqual(isSupportedV2);
-        });
-      }
-    }
-  };
-
-  compareChains('mainnet');
-  //compareChains('testnet');
-});
+//describe('compare isSupportedChain between v1 and v2 routes', () => {
```

Comment:
@kev1n-peters are going to need this test in the future? Can we remove it or revisit to re-enable it later on?

Reply:
I think we could leave the test but add a case that skips the 'mantle' chain for now. The current version of sdkv2 installed doesn't support mantle and bumping the version to latest (that supports mantle) causes all sorts of dependency and build issues that aren't worth solving in this PR, especially since the ongoing migration to use sdkv2 is going to be changing these dependencies.